docs/01-overview/_sidebar.md

```diff
@@ -6,5 +6,5 @@
   * [Observability](/01-overview/observability/README.md)
   * [Serverless](/01-overview/serverless/README.md)
   * [Service Management](/01-overview/service-management/README.md)
-  * [Service Mesh](/01-overview/service-mesh/README.md)
   * [User Interfaces](/01-overview/ui/README.md)
+  * [Kyma Istio Operator](/istio/user/00-overview/README.md)
```

Comment:
```suggestion
  * [Istio](/istio/user/00-overview/README.md)
```
Again, in the listing of components/functionality I would suggest to simply use "Istio" to not confuse the user